site/rtl.css

```
@@ -0,0 +1,813 @@
+html[dir="rtl"] .toc-status {
+  margin-right: 0;
+  margin-left: 6px;
+}
+
+html[dir="rtl"] .masthead-btn-count {
+  padding-left: 0;
+  padding-right: 8px;
+  margin-left: 0;
+  margin-right: 4px;
+  border-left: none;
+  border-right: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)